--- check_adaptec_raid	2013-04-05 11:37:33.691226127 +0200
+++ check_adaptec_raid_cat	2013-04-05 11:39:51.639223109 +0200
@@ -64,7 +64,7 @@
 	my $zmm = $_[4];
 	my $status = 0; # Return Status
 	my $statusMessage = "Controller Status "; # Return String
-	my @output = `$sudo $arcconf GETCONFIG $controller AD`; #seems to parse every line in its own variable
+     my @output = `cat /home/fnemeth/git/check_adaptec_raid/arcconf_output/controller_output`
 	my %hash;
 	my @linevalues;
 	
@@ -361,7 +361,7 @@
 		}
 		unless ( -e $arcconf ) { print "Executable not found!\n"; die; }
 	}
-	#getControllerCfg($sudo, $arcconf, $controller, $temperature, $zmm);
-	#getPhysDevCfg($sudo, $arcconf, $controller, @physDevices);
+	getControllerCfg($sudo, $arcconf, $controller, $temperature, $zmm);
+	getPhysDevCfg($sudo, $arcconf, $controller, @physDevices);
 	getLogDevCfg($sudo, $arcconf, $controller, @logDevices);
 }
